src/linear_algebra/quadratic_form.lean

```diff
@@ -816,3 +817,135 @@ theorem exists_orthogonal_basis [hK : invertible (2 : K)]
 let ⟨v, hv₁, hv₂, _⟩ := exists_orthogonal_basis' hB₁ hB₂ in ⟨v, hv₁, hv₂⟩
 
 end bilin_form
+
+/-- Nonzero elements in a field are invertible. This is a definition since it requires a
+parameter. -/
+def field.invertible {K : Type*} [field K] {z : K} (hz : z ≠ 0) : invertible z :=
```

Comment:
Does library_search find this?

Reply:
I think we have it already for group_with_zero.

Reply:
Yep, it turns out to be `invertible_of_nonzero`